src/py.py

```diff
@@ -0,0 +1,10 @@
+# shim for pylib going away
+# if pylib is installed this file will get skipped
+# (`py/__init__.py` has higher precedence)
+import sys
+
+import _pytest._py.error as error
+import _pytest._py.path as path
+
+sys.modules["py.error"] = error
```

Comment:
Thanks i hate it, but it's a fun hack

Reply:
It is, but the overreaching hack into the python import system to be backward compatible in this "yikes this works" way is quite something